app/assets/stylesheets/login.scss

```diff
@@ -0,0 +1,260 @@
+@import 'bootstrap';
+
+html, body {
+  top: 0;
+  margin: 0;
+  padding: 0;
+  background-color: white;
+  height: 100%;
+  width: 100%;
+  font-family: Verdana, sans-serif;
+}
+
+body {
+  position: absolute;
+  background: url(commit.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+p.alert {
+  color: white;
+  font-weight: bold;
+  text-shadow: 0px 0px 10px #000;
+}
+
+div.container {
+  transform: translate(-50%,-50%);
+  position: absolute;
+  top: 50%;
+  left: 50%;
+}
+
+div.field {
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+input#user_email, input#user_password {
+  background-color: rgba(0, 0, 0, 0);
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 2px solid #feb600;// #484848;
```

Comment:
Lol you might want to remove these old comments

Reply:
And, would be nice if you can abstract these colors into a variable. https://sass-lang.com/guide (check out the Variables section)